### DISCUSSION TITLE
Versions 1.1.1 and 1.1.2 are being reported as malicious

### DISCUSSION THREAD

**Discussion by dimpet19** [vincenzopalazzo/material-ui-swing#169] [General]
F-Secure and others are reporting that version 1.1.1 and 1.1.2 are malicious

![image](https://user-images.githubusercontent.com/49181700/128286166-90357507-4895-4079-ad38-562ef0ba969f.png)

![image](https://user-images.githubusercontent.com/49181700/128286412-a0ee31e1-a941-4043-bc15-aaf9bee750ba.png)

https://www.virustotal.com/old-browsers/file/b8c42d0fd7816ec188fdd5bcf912274dae00eacdc02fa84f5c111d7f7ca80b57

![image-20210803-004711](https://user-images.githubusercontent.com/49181700/128286634-60fc7506-e98f-4dc8-8376-5cf0987827c1.png)

**Comment by vincenzopalazzo** [comment]
Hi @dimpet19, sorry for that.

I need to understand what these screenshots mean and I will try to propose a solution in the next release.

I will put it in my queue.
